package/gluon-config-mode-theme/sass/gluon.scss

```diff
@@ -284,6 +284,12 @@ code {
   margin-bottom: 2em;
 }
 
+.gluon-osm-map {
+  width: 100%;
+  height: 40em;
```

Comment:
40em seems to much maybe 25 is better the map looks nicer.

Reply:
The width of the main content scales between 40em and 60em (depending on the window width), meaning that with a height of 40em the map is square in narrow windows, and has a 3:2 aspect ratio in wide windows. This is already close to the usual widescreen display aspect ratio of 16:10.

Given that the relevant areas for sites should be circular on average, a square seems like the best approximation to me. Further decreasing the height would make the map less square, so we should either increase the height, or keep the current value...